app/lib/publishing_api_v2.rb

```diff
@@ -0,0 +1,10 @@
+require "gds_api/publishing_api_v2"
+
+class PublishingApiV2
+  def self.instance
+    GdsApi::PublishingApiV2.new(
```

Comment:
I'd be 100% surprised if we need `publishing_api` and `publishing_api_v2`.  I suspect we can drop the former, and rename the latter to `publishing_api`